### PR TITLE
[dpc++][pstl] shift_left, shift_right (C++20) mention

### PR DESCRIPTION
**Pull request by MikeDvorskiy**
[dpc++][pstl] + shift_left, shift_right (C++20) mention

### CODE REVIEW

**Review by capatober — DISMISSED** [review]
I don't have any comments here

**Review by capatober — DISMISSED** [review]

File: documentation/library_guide/pstl_main.rst [hunk 1]
```diff
@@ -4,11 +4,11 @@ Parallel STL Overview
...
 =============================
 
-Parallel STL is an implementation of the C++ standard library algorithms with support
-for execution policies, as specified in ISO/IEC 14882:2017 standard, commonly called C++17.
-The implementation also supports the unsequenced execution policy specified in the
-final draft for the C++ 20 standard (N4860). For more details see `the standard execution policies
-<https://en.cppreference.com/w/cpp/algorithm/execution_policy_tag_t>`_.
+Parallel STL is an implementation of the C++ standard library algorithms with support for execution
+policies, as specified in ISO/IEC 14882:2017 standard, commonly called C++17. The implementation also
+supports the unsequenced execution policy and a couple of additional algorithms - ``shift_left``, ``shift_right``,
```
Comment (dcbenito) [inline documentation/library_guide/pstl_main.rst:9]: and the algorithms ``shift_left`` and ``shift_right``,

File: documentation/library_guide/pstl_main.rst [hunk 2]
```diff
@@ -4,11 +4,11 @@ Parallel STL Overview
...
+specified in the final draft for the C++ 20 standard (N4860). For more details see `the standard execution
```
Comment (dcbenito) [inline documentation/library_guide/pstl_main.rst:10]: Please change: [specified in the] to [which are specified in the]

**Review by dcbenito — CHANGES_REQUESTED** [review]
Please see inline comments, and add the updates to the 'release/2021.2' branch so the changes appear for the release.

@AlexVeprev is my above request correct?

**Comment by AlexVeprev** [comment]
> Please see inline comments, and add the updates to the 'release/2021.2' branch so the changes appear for the release.
> @AlexVeprev is my above request correct?

I'm not tracking release process closely. As far as I know 2021.2 has already been published, so I'd not expect updates there, @ValentinaKats could you please comment?

**Comment by ValentinaKats** [comment]
> > Please see inline comments, and add the updates to the 'release/2021.2' branch so the changes appear for the release.
> > @AlexVeprev is my above request correct?
> 
> I'm not tracking release process closely. As far as I know 2021.2 has already been published, so I'd not expect updates there, @ValentinaKats could you please comment?

All proposed changes were timely added to 'release/2021.2' branch and were published as part of 2021.2 release documentation. No issues here. I'm closing this PR as these changes were cherry picked from 'release/2021.2' into 'main' in scope of PR #186.

**Comment by dcbenito** [comment]
Thank you, @ValentinaKats!

**Pull request closed without merging**